generators/entity-server/templates/src/main/resources/config/liquibase/changelog/added_entity.xml.ejs

```diff
@@ -217,7 +217,11 @@
             <%_ } _%>
             <%_ for (idx in relationships) {
                     let loadColumnType = 'numeric';
-                    if (relationships[idx].relationshipValidate === true && relationships[idx].relationshipRequired) {
+                    if (relationships[idx].relationshipValidate === true && relationships[idx].relationshipRequired
+                        && (relationshipType === "many-to-one"
```

Comment:
@murdos you are using in the for loop always the same `relationshipType`(line221); you will need at least ` relationshipType = relationships[idx].relationshipType;`

Reply:
Oh, right! 🙃 Thanks for catching this!
I'll fix it right now...

Reply:
In order to be able to test, I added in my repo and run the generator. With that fix is working.

Reply:
I ran tests locally too, and confirmed the issue you reported is fixed.